breaking: remove `experimental.handleRenderingErrors` flag (#16265)
sveltejs/kit: documentation/docs/30-advanced/25-errors.md

	const post = await db.getPost(params.slug);

	if (!post) {
		error(404, 'Not found');
	}

	return { post };
}
```

> This throws an exception that SvelteKit catches, causing it to set the response status code to 404 and render an [`+error.svelte`](routing#error) component, where the `error` is the object provided as the second argument to `error(...)`.

Author: ```suggestion
This throws an exception that SvelteKit catches, causing it to set the response status code to 404 and render an [`+error.svelte`](routing#error) component, where the `error` is an `App.Error` object with the provided `status` and `message`.
```

```svelte
<!--- file: src/routes/+error.svelte --->
<script>

> 	let { error } = $props();

Reviewer: Flagging for later; is this correct? The text of the paragraph is misleading -- it says "`error` is the object provided as the second argument to `error(...)`", but here the second argument is a string, so `error.message` below _should_ throw.

Author: the wording above was incorrect, now fixed

// @filename: index.js
import { error } from '@sveltejs/kit';
// ---cut---
error(404, {
	message: 'Not found',
error(404, 'Not found', {
	+++code: 'NOT_FOUND'+++
});
```

...otherwise, for convenience, you can pass a string as the second argument:

```js
import { error } from '@sveltejs/kit';
// ---cut---
---error(404, { message: 'Not found' });---
+++error(404, 'Not found');+++
```

> 

Reviewer: Not really sure why you edited the way you did -- needs a larger overhaul to make sense.

Author: Passing an object as the second argument is deprecated. The signature is now basically

```js
error(status: number, message?: string, extra?: Omit<App.Error, 'status' | 'message'>)
```
